src/overlays/actors/ovl_En_Light/z_en_light.c

```diff
@@ -180,7 +179,7 @@ void EnLight_Draw(Actor* thisx, GlobalContext* globalCtx) {
                    Gfx_TwoTexScroll(globalCtx->state.gfxCtx, 0, 0, 0, 16, 32, 1, ((this->timer * 2) & 63),
                                     (this->timer * -6) & 127 * 1, 16, 32));
 
-        dList = D_05000440;
+        dList = gUnusedCandleDL;
```

Comment:
This unused candle seems to actually be used, right?

Reply:
From what I saw in actor lists these params are never used like this.